{
  "plugin": {
    "conspiracy": {
      "command": {
        "ping": {
          "name": "ping",
          "description": "Ping Pong üèì",
          "response": "Pong üèì"
        }
      }
    },
    "default": {
      "command": {
        "invite": {
          "name": "invite",
          "description": "Invite this bot to your server!",
          "response": "Here you go üéâ",
          "buttonText": "Invite this bot"
        },
        "language": {
          "name": "language",
          "description": "Set the language this bot uses on this server",
          "option": {
            "language": {
              "name": "language",
              "description": "Choose a language for this server",
              "choice": {
                "de": "German",
                "en": "English"
              }
            }
          },
          "same": "Hey, looks like everything will stay the same ü§£",
          "response": "Changed the language from `{{old}}` to `{{new}}`. It may take a while for the change to be completed.",
          "locale": {
            "de": "German",
            "en": "English",
            "none": "none"
          }
        }
      }
    }
  },
  "constants": {
    "messages": {
      "COMMAND_INTERNAL_ERROR": "An error ocurred, pleasy try again later!",
      "COMMAND_NOT_FOUND": "The user couldn't be found!",
      "COMMAND_USER_NOT_FOUND": "constants.messages.COMMAND_USER_NOT_FOUND",
      "COMMAND_TRY_AGAIN": "I'm working right now, pleasy try again shortly!"
    }
  }
}
